src/dto/FactionDto.js

```diff
@@ -0,0 +1,35 @@
+/** Class representing faction data.
+ * Note that this does not contain constant faction data.
+ * @see {@link Factions}
+ */
+export default class FactionDto {
+    /**
+     * Create new faction data.
+     * @param {String[]} awards - The faction's awards.
+     * @param {Object<string, number>} items - The faction's items.
+     */
+    constructor(awards = [], items = {}) {
```

Comment:
If you're making this, we can abandon the Factions file completely and put everything in here instead.

Reply:
Well, the constants aren't values changed that often. These very well could change via a developer command and such. Also, these are less important to move from bot to bot, whereas if you were to move data such as its fullName, channels, emote, joinMessages to the database it would break any new version of the bot that someone runs.